packages/squiggle-lang/src/fr/dist.ts

```diff
@@ -245,15 +245,26 @@ export const library: FRFunction[] = [
   ...["to", "credibleIntervalToDistribution"].map((name) =>
     maker.make({
       name,
-      examples: name === "to" ? ["5 to 10", "to(5,10)", "-5 to 5"] : [],
+      examples: name === "to" ? ["5 to 10", "to(5,10)"] : [],
       definitions: [
-        makeTwoArgsDist((low, high) =>
-          SymbolicDist.makeFromCredibleInterval({
+        makeTwoArgsDist((low, high) => {
+          if (low >= high) {
+            throw new REDistributionError(
+              argumentError("Low value must be less than high value")
+            );
+          } else if (low <= 0 || high <= 0) {
+            throw new REDistributionError(
+              argumentError(
+                `The "to" function only accepts paramaters above 0. It's a shorthand for lognormal({p5:min, p95:max}), which is only valid with positive entries for then minimum and maximum. If you would like to use a normal distribution, which accepts values under 0, you can use it like this: normal({p5:${low}, p95:${high}).`
```

Comment:
Great error message!

There's a missing `}`:

```suggestion
                `The "to" function only accepts paramaters above 0. It's a shorthand for lognormal({p5:min, p95:max}), which is only valid with positive entries for then minimum and maximum. If you would like to use a normal distribution, which accepts values under 0, you can use it like this: normal({p5:${low}, p95:${high}}).`
```

Reply:
Thanks for flagging! Will fix!